builtin/providers/ns1/resource_apikey.go

```diff
@@ -3,7 +3,7 @@ package ns1
 import (
 	"github.com/hashicorp/terraform/helper/schema"
 
-	nsone "gopkg.in/ns1/ns1-go.v2/rest"
+	ns1 "gopkg.in/ns1/ns1-go.v2/rest"
```

Comment:
I don't love importing into `ns1` while in a packaged named `ns1`. But there are worse things in like, and at least this reads cleanly when you grok it. Hopefully new readers of this code will realize what's going on.